src/db.py

```diff
@@ -46,6 +46,14 @@ def set_user_access_token(netid, access_token):
 	"""
 	mongo.db.users.update({"_id": netid}, {"access_token": access_token}, upsert=True)
 
+def get_courses_for_student_or_staff(netid):
+	student_course = get_courses_for_student(netid)
+	staff_course = get_courses_for_staff(netid)
+	for course in staff_course:
+		if course not in student_course:
+			student_course.append(course)
```

Comment:
This is a simple O(n^2) algo that merges two sets. I don't think a person will be in that many courses to cause performance issue here. Couldn't use set because `course` is a dict and is un-hashable.